sp800-63c/sec6_presentation.md

```diff
@@ -6,6 +6,8 @@ Assertions MAY be presented in either a *back-channel* or *front-channel* manner
 
 The IdP SHALL transmit only those attributes that were explicitly requested by the RP. RPs SHALL conduct a privacy risk assessment when determining which attributes to request. 
 
+To mitigate the risk of unauthorized exposure of sensitive information (e.g., shoulder surfing), the CSP SHALL, by default, mask sensitive information displayed to the subscriber.  The CSP SHALL provide mechanisms for the subscriber to temporarily unmask such information in order for the subscriber to view full values.
```

Comment:
We don't have a definition of what it means to "mask" information. For example, we should allow the CSP to "mask" by not displaying the information in the first place, and the current language choice sounds like it's only black-out.